libdokan/fs.go

```diff
@@ -377,11 +377,11 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 			return nil, false, dokan.ErrAccessDenied
 		}
 		return f.root.private.open(ctx, oc, ps[1:])
-	case `TEAM` == ps[0]:
+	case strings.ToUpper(TeamName) == ps[0]:
```

Comment:
Do you think the Go compiler will optimize out this `ToUpper` call on a constant?